src/google/protobuf/compiler/command_line_interface.cc

```diff
@@ -2421,7 +2421,7 @@ void CommandLineInterface::PrintHelpText() {
   std::cout << "Usage: " << executable_name_ << " [OPTION] PROTO_FILES";
   std::cout << R"(
 Parse PROTO_FILES and generate output based on the options given:
-  -IPATH, --proto_path=PATH   Specify the directory in which to search for
+  -IPATH, -I, --proto_path=PATH   Specify the directory in which to search for
                               imports.  May be specified multiple times;
```

Comment:
Happy to fix all this indentation formatting too - hope to get a preliminary 👍 from a reviewer first before proceeding.

Reply:
I was under the impression that -IPATH already indicated the ability to use -I in place of --proto_path